test/functional/apps/visualize/_tsvb_chart.js

```diff
@@ -28,28 +28,56 @@ export default function ({ getService, getPageObjects }) {
         .then(function clickMetric() {
           return PageObjects.visualBuilder.clickMetric();
         })
+        .then(function addSeries() {
+          return PageObjects.visualBuilder.addSeries();
+        })
+        .then(function addMetric() {
+          return PageObjects.visualBuilder.addMetric();
+        })
+        .then(function selectOverallAverage() {
+          return PageObjects.visualBuilder.selectItem('.vis_editor__series:last-child ' +
+          '.vis_editor__metric:last-child .vis_editor__aggregation',
+          'Overall Sum');
+        })
+        .then(function selectCountMetric() {
+          return PageObjects.visualBuilder.selectItem('.vis_editor__series:last-child ' +
+          '.vis_editor__metric:last-child .vis_editor__std_sibling-metric',
+          'Count');
+        })
         .then(function sleep() {
           return PageObjects.common.sleep(1003);
         });
     });
 
 
+
     describe('Visual Builder chart', function indexPatternCreation() {
```

Comment:
I know this wasn't a change in this PR, but could you change the describe on line 7 to `Visual Builder`.   You'll notice in Jenkins output when the before method failed in that last run that the error has `Visualize app` (from index.js) and `Visualize app` from the describe on line 7 which doesn't help someone debugging know which test it is.

Also, the function in both of the describes have incorrect names of `describeIndexTests` and `indexPatternCreation` which aren't correct and could just be removed (they don't show up in the stack trace when it failed).

Reply:
Yeah... That looks like a copy and paste job. I re-named everything and moved it around a bit so it makes room for future tests.

Reply:
Input controls has made a bit of progress in this area by creating some useful test functions when dealing with react-select elements.  Check out the *reactSelect* functions starting here:

https://github.com/elastic/kibana/blob/master/test/functional/page_objects/visualize_page.js#L175

Might help simplify the CSS selectors and make it a little easier to read.  Unfortunately, no way we've been able to figure out yet for how to use the data-test-subj field on the options since react-select doesn't seem to offer the ability to pass in custom html attributes for each option.